### ISSUE TITLE
Fine tune the Stability of Video Detection?

### ISSUE THREAD

**Issue by Dawson-huang** [ceykmc/video_object_detection_metrics#1]
Can I use these three errors as loss functions to fine-tune the video detect model? 
If allowed to do as loss functions, could you tell me how to do it bester?
Thank you very mach!

**Comment by ceykmc** [comment]
I am afraid we can not use these three errors as loss function to fine-tune the video detect model directly.
In the paper, they just used weighted nms, motion guided propagation and object tracking to improve stability.

**Comment by Dawson-huang** [comment]
Ok, I get it.
Thank you very much for your reply!
dawson
